crates/jstzd/src/jstzd.rs

```diff
@@ -0,0 +1 @@
+pub const DEFAULT_RPC_ENDPOINT: &str = "localhost:8732";
```

Comment:
This doesn't seem to be referenced anywhere?

Reply:
used as a default value for the node endpoint when building the octez_client